caddytls/config.go

```diff
@@ -532,12 +532,14 @@ var supportedKeyTypes = map[string]acme.KeyType{
 
 // Map of supported protocols.
 // HTTP/2 only supports TLS 1.2 and higher.
-var supportedProtocols = map[string]uint16{
+var SupportedProtocols = map[string]uint16{
 	"tls1.0": tls.VersionTLS10,
 	"tls1.1": tls.VersionTLS11,
 	"tls1.2": tls.VersionTLS12,
 }
 
+// NOTE: if updating the above map, also update sslProtocolToStringMap in caddyhttp/fastcgi/fastcgi.go
```

Comment:
Put this in the godoc of SupportedProtocols instead. Preferably we would just use one map though. Why not just use this one?

Reply:
Yes, one map would be better, but the format used in Caddyfile (for the tls settings) is different from the format used by fastcgi/mod_ssl, which is why two maps are required (otherwise I would have used what was already there).

Reply:
That's fine then -- thanks. But move it to the godoc for that map, rather than floating by itself